src/Validator/ErrorElement.php

```diff
@@ -221,10 +246,13 @@ private function newConstraint(string $name, array $options = [])
         return new $className($options);
     }
 
-    private function getCurrentPropertyPath(): ?PropertyPath
+    /**
+     * @return PropertyPath|null
+     */
+    protected function getCurrentPropertyPath()
     {
         if (!isset($this->propertyPaths[$this->current])) {
-            return null; //global error
+            return; //global error
```

Comment:
Even if it was previously written in this way, it is wrong. If this function is able to return a value, it must return `null` here.

Reply:
@phansys fix. I add `@final` phpdoc too.